diff --git a/krita_image_search/krita_image_docker.py b/krita_image_search/krita_image_docker.py
index d9e4002..77510de 100644
--- a/krita_image_search/krita_image_docker.py
+++ b/krita_image_search/krita_image_docker.py
@@ -2,7 +2,7 @@ from PyQt5.QtWidgets import QLabel, QLineEdit, QWidget, QScrollArea, QVBoxLayout
 from PyQt5.QtCore import Qt, QObject, QThread, QByteArray, pyqtSignal, QRect, QSize, QMargins, QPoint
 from PyQt5.QtGui import QMovie, QPixmap
 from krita import *
-import aiohttp
+from krita_image_search.vendor import aiohttp
 import asyncio
 from .resources import *
 import socket
